pkgs/by-name/mo/moneydance/package.nix

```diff
@@ -20,36 +21,48 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-wwSb3CuhuXB4I9jq+TpLPbd1k9UzqQbAaZkGKgi+nns=";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [
+    makeWrapper
+    wrapGAppsHook3
```

Comment:
`(buildPackages.wrapGAppsHook3.override { makeWrapper = buildPackages.makeShellWrapper; })`

Reply:
@Artturin My hero! Applied this and verified it does the right thing.